pcap-remote.c

```diff
@@ -513,12 +513,14 @@ static int pcap_stats_remote(pcap_t *p, struct pcap_stat *ps)
  * \brief It retrieves network statistics from the other peer.
  *
  * This function is just a void cointainer, since the work is done by the rpcap_stats_remote().
- * See that funcion for more details.
+ * See that function for more details.
  *
  * Parameters and return values are exactly the same of the pcap_stats_ex().
  */
-static struct pcap_stat *pcap_stats_ex_remote(pcap_t *p)
+static struct pcap_stat *pcap_stats_ex_remote(pcap_t *p, int *pcap_stat_size)
 {
+	*pcap_stat_size = sizeof(p->stat);
```

Comment:
I checked in a fix for that particular issue in eeee5694eb0620ec38423700d7a9e7bd26c87dd2.